lib/chat_api/conversations/conversation.ex

```diff
@@ -87,15 +87,23 @@ defmodule ChatApi.Conversations.Conversation do
     |> cast(attrs, [:last_activity_at])
   end
 
-  defp put_closed_at(%Ecto.Changeset{valid?: true, changes: %{status: status}} = changeset) do
+  defp put_closed_and_last_activity_at(
+         %Ecto.Changeset{valid?: true, changes: %{status: status}} = changeset
+       ) do
+    now = DateTime.utc_now() |> DateTime.truncate(:second)
```

Comment:
hmm i know we do this in some other places but i forget why we truncate by `:second` here... do you happen to know?

Reply:
Nope, I was curious too. It was there and I left it in. I can remove it – I don't really see any reason for it?

Reply:
yeah maybe just remove it and see if it breaks anything 🤷 but if it doesn't, let's avoid truncating

Reply:
oh lol this is why 

<img width="988" alt="image" src="https://user-images.githubusercontent.com/7440689/115478075-4d4e7d80-a213-11eb-8afa-f669d752d5e4.png">

Reply:
ahhh good to know!